ArcGISRuntimeSDKQt_CppSamples/Search/FindPlace/FindPlace.h

```diff
@@ -74,6 +75,7 @@ class FindPlace : public QQuickItem
   bool poiTextHasFocus() const { return m_poiTextHasFocus; }
   void setPoiTextHasFocus(bool hasFocus);
   Esri::ArcGISRuntime::GeocodeParameters createParameters();
+  void geocodeCompleteHandler(const QList<Esri::ArcGISRuntime::GeocodeResult>& results);
```

Comment:
This is subjective, but I would go with `onGeocodingCompleted_`. Our pattern has been to suffix private functions with a trailing underscore, but more so I think "onSomethingHappened_" naming convention reads nicer IMO.

The reason is that "handler" sounds more like a slot; something you connect directly to a signal. In these cases, you are calling a function explicitly.  What are your thoughts?

Reply:
I agree. Modified all to "onSomethingHappened_" format.